spec/connect/client_spec.rb

```diff
@@ -5,7 +5,12 @@
   let(:default_logger) { SUSE::Connect::GlobalLogger.instance.log }
   let(:string_logger) { ::Logger.new(StringIO.new) }
   let(:client_instance) { described_class.new config }
-  let(:product) { SUSE::Connect::Remote::Product.new identifier: 'SLES', version: '12', arch: 'x86_64' }
+
+  let(:product_tree) { JSON.parse(File.read('spec/fixtures/product_tree.json')) }
+  let(:product) { Remote::Product.new(product_tree) }
+
+  let(:submodule) { product.extensions[1] }
+  let(:subsubmodule) { product.extensions[1].extensions[1] }
```

Comment:
This is a `submodule`. Also `let(:submodule) { module.extensions[1] }` would maybe have a bit nicer semantics.